content/docs/2.16/scalers/redis.md

```diff
@@ -1,15 +1,15 @@
 +++
-title = "Redis Lists"
+title = "Redis Lists Or Strings"
```

Comment:
I renamed redis-lists.md to redis.md because there is more than just a list, is this acceptable?

Reply:
I think that this could break the current naming of the urls. @zroubalik @wozniakjan ?

Reply:
If change, then I would probably go with just `Redis`

Do you mean breaking urls between versions of this scaler, or what? Then we can probably try to find a solution or just keep the original name.